entries/z/zillow.com.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Zillow": {
+    "domain": "zillow.com",
+    "tfa": [
+      "sms",
+      "call"
```

Comment:
I don't see any mention of calls for 2FA mentioned on the documentation page you linked.

Reply:
Strange. It is an option in account settings though:

![image](https://github.com/user-attachments/assets/9fbb7eb9-4fb5-4976-b95d-503acdab7d8f)